android/lib/pages/register.dart

```diff
@@ -105,73 +105,6 @@ class _RegisterState extends State<Register> {
       ),
     ));
 
-
-    final nameField = inputField(TextFormField(
-      onSaved: (value) => _name = value,
-      autofocus: false,
-      decoration: const InputDecoration(
-        border: InputBorder.none,
-        hintText: 'Name (optional)',
-      ),
-    ));
-
-    final surnameField = inputField(TextFormField(
-      onSaved: (value) => _surname = value,
-      autofocus: false,
-      decoration: const InputDecoration(
-        border: InputBorder.none,
-        hintText: 'Surname (optional)',
-      ),
-    ));
-
-    final ageField = inputField(TextFormField(
-      keyboardType: TextInputType.number,
-      autofocus: false,
-      inputFormatters: [LengthLimitingTextInputFormatter(2)],
-      decoration: const InputDecoration(
-        border: InputBorder.none,
-        hintText: 'Age (optional)',
-      ),
-    ));
-
-    final countryField = inputField(GestureDetector(
```

Comment:
Yes, we are removing those, but let us not forget our implementation of the Country Picker. We may use this widget in the settings page, too.